Tests/PubNubTests/Mocking/MockURLSession.swift

```diff
@@ -71,60 +71,144 @@ class MockURLSessionTask: URLSessionDataTask {
   }
 }
 
+class MockURLSessionUploadTask: URLSessionUploadTask {
+  var mockSession: URLSessionReplaceable
+  
```

Comment:
Codacy found an issue: [Lines should not have trailing whitespace.](https://app.codacy.com/manual/PubNub/swift/pullRequest?prid=6302911)